docs/sources/dashboards/time-range-controls.md

```diff
@@ -110,7 +110,7 @@ For more advanced time settings, click the **Dashboard settings** (gear) icon at
 
 ## Panel time overrides and timeshift
 
-In [Query options]({{< relref "../panels/queries.md#query-options" >}}), you can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the upper right. This allows you to show metrics from different time periods or days at the same time.
+In [Query options]({{< relref "../panels/queries.md#query-options" >}}), you can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the upper right. This allows you to show metrics from different time periods or days at the same time. This will only work if the main timepicker is using a relative time range. If you are using an absolute time range, that absolute range will be used instead. 
```

Comment:
```suggestion
In [Query options]({{< relref "../panels/queries.md#query-options" >}}), you can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the upper right. This allows you to show metrics from different time periods or days at the same time. It will only work if the main time picker is using a relative time range. If you are using an absolute time range, that absolute range will be used instead. 
```